src/lib.rs

```diff
@@ -229,11 +230,12 @@ pub fn get_public_key(
 /// To convert the answer into `near_crypto::Signature` do:
 ///
 /// ```
-/// near_crypto::Signature::from_parts(near_crypto::KeyType::ED25519, &signature)
-///     .expect("Signature is not expected to fail on deserialization")
+/// # let signature = [10; 64].to_vec();
+/// let signature = near_crypto::Signature::from_parts(near_crypto::KeyType::ED25519, &signature)
+///     .expect("Signature is not expected to fail on deserialization");
 /// ```
 pub fn sign_transaction(
-    unsigned_transaction_borsh_serializer: BorshSerializedUnsignedTransaction,
+    target: SignTarget,
```

Comment:
To be honest, from a crate user perspective, I would prefer to have a single `sign_transaction` method that would figure out when it should use blind signing or not, and maybe additionally have 3 lower-level methods:

* non-blind transaction signing (receive a serialized transaction)
* blind transaction signing (receive a serialized transaction)
* blind transaction hash signing (receive a hash)